packages/components/src/lib/MarkdownViewer.tsx

```diff
@@ -47,44 +79,59 @@ export const MarkdownViewer: React.FC<MarkdownViewerProps> = ({
         textColor || "prose-stone",
         textSize === "sm" ? "text-sm" : "text-xs"
       )}
-      rehypePlugins={[rehypeInlineCodeProperty]}
+      rehypePlugins={[rehypeInlineCodeProperty, rehypeRaw]}
```

Comment:
This is dangerous! It allows any html in `@doc` tags.

Example: https://squiggle-website-kh5mxysfa-quantified-uncertainty.vercel.app/playground?v=dev#code=eNotjcsKgzAQRX9lyEaFSpXSTXwg9AP6A25SM5HQNBPiaF%2F032uly3MPl%2FMWGo2aHZ9Io5Ci0zSkSa3tAhM%2FHTa9CDRZtuQlRHSK7YIVMAUJeVkU4VGBQ8MSyuMGhjznk32hPGw8kKP4u%2BoKLmq4jpFmr%2FP%2FPEZE34uWFozG0b3er%2BU2yXrfTawin8OqvYIGPMWbcmmxgzITny%2BBoD5H

It sanitizes html somewhat (I couldn't do `<script>alert(1)</script>` or `<button onclick="javascript:alert(1)">`), but I don't think we can rely on it being secure.